src/cert/export.rs

```diff
@@ -17,13 +22,23 @@ fn identify_cert(buf: &[u8]) -> EncodingFormat {
     }
 }
 
+// Convert kernel formatted certs into user readable certificates
+fn cert_entries(cert_bytes: Vec<u8>) -> Result<Vec<CertTableEntry>> {
+    let certs = CertTableEntry::vec_bytes_to_cert_table(cert_bytes.to_vec())
+        .context("Could not convert bytes to certs")?;
+    Ok(certs)
+}
```

Comment:
Any reason we are doing a move (copy) of cert_bytes for each step here? Would it make sense to do something like this:

```suggestion
fn cert_entries(cert_bytes: &[u8]) -> Result<Vec<CertTableEntry>> {
    let certs = CertTableEntry::vec_bytes_to_cert_table(cert_bytes)
        .context("Could not convert bytes to certs")?;
    Ok(certs)
}
```